docs/examples/move/nft_marketplace/README.md

```diff
@@ -0,0 +1,179 @@
+# Marketplace Guide
+
+The `marketplace_extension.move` module provides a straightforward implementation of a marketplace extension. To utilize it, follow the steps outlined below.
+The `item_for_market.move` contains mocked item data for use within the marketplace.
+The `rental_extension.move` is an extension adds functionality to enable item rentals.
+
+## Steps to Use the Marketplace
+
+### 1. Connect to the Network
+
+Connect to the IOTA network (e.g., using a faucet to obtain tokens).
+
+### 2. Install Kiosk
+
+By installation, we mean creating a Kiosk object and an OwnerCap, then transferring them to the caller.
+Run the following command to install the Kiosk module:
+
+```bash
+iota client call \
+    --package 0x2 \
+    --module kiosk \
+    --function default
+```
+
+### 3. Publish `nft_marketplace` package
+
+#### 3.1(Optional) Publish Kiosk rules modules if these are not present in the network you are using
+
+Publish Kiosk rules modules(package):
+
+```bash
+iota client publish iota/kiosk
+```
+
+After publishing, export the following variable:
+
+- `RULES_PACKAGE_ID`: The ID of rules package.
+
+#### 3.2 Publish marketplace extension
+
+Publish the marketplace_extension.move module:
+
+```bash
+iota client publish iota/docs/examples/move/nft_marketplace
+```
+
+After publishing, export the following variables:
+
+- `MARKETPLACE_PACKAGE_ID`: The ID of whole marketplace package.
+- `MARKETPLACE_PUBLISHER_ID`: The ID of the publisher object created during marketplace package publishing."
+
+### 4. Create an Clothing Store Item
+
+Create an item, for instance:
+
+```bash
+iota client call \
+    --package $MARKETPLACE_PACKAGE_ID \
+    --module clothing_store \
+    --function new_jeans
+```
+
+After creation, export the following variable:
+
+- `CLOTHING_STORE_ITEM_ID`: The ID of the published item (in this case, Jeans).
+
+### 5. Create a Transfer Policy
+
+`TransferPolicy` is a generic shared object acting as a central authority enforcing everyone to check their purchase is valid against the defined policy before the purchased item is transferred to the buyers. Object is specified by concrete type.
+`default` function creates `TransferPolicy` object and an OwnerCap, then transferring them to the caller.
```

Comment:
Should it be:
 a `TransferPolicy` object and a `TransferPolicyCap`?

Reply:
it’s a description of `TransferPolicy` itself we don’t have to focus on `TransferPolicyCap`
But below, in the description I’ve admitted that we have to import `TransferPolicyCap` too

Reply:
It is not clear what the `OwnerCap` is. It is better to clarify because I can't find such a type or explanation to this entity.

Reply:
Short description of `TransferPolicyCap` has been added

Reply:
Why did you leave the `OwnerCap`?

Reply:
Fixed